src/rpc/rawtransaction.cpp

```diff
@@ -345,24 +364,55 @@ UniValue getrawtransaction(const UniValue& params, bool fHelp)
             "         \"hex\"                     (string) output note commitment\n"
             "         ,...\n"
             "       ],\n"
-            "       \"onetimePubKey\" : \"hex\",  (string) the onetime public key used to encrypt the ciphertexts\n"
-            "       \"randomSeed\" : \"hex\",     (string) the random seed\n"
+            "       \"onetimePubKey\" : \"hex\",    (string) the onetime public key used to encrypt the ciphertexts\n"
+            "       \"randomSeed\" : \"hex\",       (string) the random seed\n"
             "       \"macs\" : [                  (json array of string)\n"
             "         \"hex\"                     (string) input note MAC\n"
             "         ,...\n"
             "       ],\n"
-            "       \"proof\" : \"hex\",          (string) the zero-knowledge proof\n"
+            "       \"proof\" : \"hex\",            (string) the zero-knowledge proof\n"
             "       \"ciphertexts\" : [           (json array of string)\n"
             "         \"hex\"                     (string) output note ciphertext\n"
             "         ,...\n"
             "       ]\n"
             "     }\n"
             "     ,...\n"
             "  ],\n"
-            "  \"blockhash\" : \"hash\",   (string) the block hash\n"
-            "  \"confirmations\" : n,      (numeric) The confirmations\n"
-            "  \"time\" : ttt,             (numeric) The transaction time in seconds since epoch (Jan 1 1970 GMT)\n"
-            "  \"blocktime\" : ttt         (numeric) The block time in seconds since epoch (Jan 1 1970 GMT)\n"
+            "  \"valueBalance\" : x.xxx,  (numeric, optional) The sapling pool balance change in " + CURRENCY_UNIT + "\n"
```

Comment:
We also have `valueBalanceZat` here for Sapling. (Both of these will be deprecated soon, replaced with a `sapling` object, so doesn't matter too much).